.github/workflows/d.yml

```diff
@@ -3,44 +3,58 @@ on: [push, pull_request]
 
 jobs:
   test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
?

Reply:
This allows using `[skip ci]` in the commit messages, which tells GitHub to skip running the CI.